static/js/util/sanctuary_test.js

```diff
@@ -4,10 +4,13 @@ import { assert } from 'chai';
 
 import { allJust, mstr } from './sanctuary';
 
-describe('sanctuary util functions', () => {
-  let maybeEquality = (j1, j2) => assert(j1.equals(j2), "just equality");
+export const maybeEquality = (j1: Maybe, j2: Maybe) => {
+  assert(j1.equals(j2), "just equality");
+};
+
+export const isNothing = (m: Maybe) => assert(m.isNothing, "should be nothing");
```

Comment:
ah yeah I agree, I'll do `isNothing -> assertIsNothing` and `maybeEquality -> assertMaybeEquality`.